### PR TITLE
adding descriptions about replication.

### PR DESCRIPTION
**Pull request by iwasakims**

### CODE REVIEW

File: hbase_regionserver/metadata.csv [hunk 1]
```diff
@@ -7,9 +7,9 @@ hbase.regionserver.ipc.total_call_time.mean,gauge,"",millisecond,"","total call
 hbase.regionserver.ipc.total_call_time.median,gauge,"",millisecond,"","total call time, including both queued and processing time.",0,hbase_regionserver,""
 hbase.regionserver.ipc.total_call_time.percentile.99,gauge,"",millisecond,"","total call time, including both queued and processing time.",0,hbase_regionserver,""
 hbase.regionserver.regions.num_regions,gauge,"","","",Number of regions in the metrics system,0,hbase_regionserver,""
-hbase.regionserver.replication.sink.applied_ops,gauge,"","","","",0,hbase_regionserver,""
-hbase.regionserver.replication.sink.age_of_last_applied_op,gauge,"","","","",0,hbase_regionserver,""
-hbase.regionserver.replication.sink.applied_batches,gauge,"","","","",0,hbase_regionserver,""
+hbase.regionserver.replication.sink.applied_ops,gauge,"","","",Number of WAL entries applied on replication sink.,0,hbase_regionserver,""
+hbase.regionserver.replication.sink.age_of_last_applied_op,gauge,"","","",Time lag of last applied WAL entry between master and slave in milliseconds.,0,hbase_regionserver,""
```
Comment (everpeace) [inline hbase_regionserver/metadata.csv]: @iwasakims  Does metrics name including "age" induce that unit of the metrics will be `millisecond`!?  If so, we could add this rule to unit detection in [`_gen.rb`](https://github.com/everpeace/integrations-extras/blob/hbase/hbase_regionserver/_gen.rb#L39-L51).

Comment (iwasakims) [reply hbase_regionserver/metadata.csv]: I think there is no convention that "age" means miliis. I just updated description as time_unit is automatically set based on the word "time".

File: hbase_regionserver/metadata.csv [hunk 2]
```diff
@@ -7,9 +7,9 @@ hbase.regionserver.ipc.total_call_time.mean,gauge,"",millisecond,"","total call
 hbase.regionserver.ipc.total_call_time.median,gauge,"",millisecond,"","total call time, including both queued and processing time.",0,hbase_regionserver,""
 hbase.regionserver.ipc.total_call_time.percentile.99,gauge,"",millisecond,"","total call time, including both queued and processing time.",0,hbase_regionserver,""
 hbase.regionserver.regions.num_regions,gauge,"","","",Number of regions in the metrics system,0,hbase_regionserver,""
-hbase.regionserver.replication.sink.applied_ops,gauge,"","","","",0,hbase_regionserver,""
-hbase.regionserver.replication.sink.age_of_last_applied_op,gauge,"","","","",0,hbase_regionserver,""
-hbase.regionserver.replication.sink.applied_batches,gauge,"","","","",0,hbase_regionserver,""
+hbase.regionserver.replication.sink.applied_ops,gauge,"","","",Number of WAL entries applied on replication sink.,0,hbase_regionserver,""
+hbase.regionserver.replication.sink.age_of_last_applied_op,gauge,"",millisecond,"",Replication time lag of last applied WAL entry between source and sink.,0,hbase_regionserver,""
```
Comment (everpeace) [inline hbase_regionserver/metadata.csv]: would you mind adding `unit_name: millisecond` in `_metrics.yml`?  `_gen.rb` supports `unit_name` in `_metrics.yml`.

File: hbase_regionserver/_metrics.yml [hunk 3]
```diff
@@ -41,14 +41,17 @@
   attribute: sink.appliedOps
   metric_type: gauge
   alias: hbase.regionserver.replication.sink.applied_ops
+  description: Number of WAL entries applied on replication sink.
 - bean: Hadoop:service=HBase,name=RegionServer,sub=Replication
```
Comment (everpeace) [inline hbase_regionserver/_metrics.yml:45]: would you mind adding `unit_name: millisecond` here too?  `_gen.rb` supports `unit_name` in `_metrics.yml`.

Comment (iwasakims) [reply hbase_regionserver/_metrics.yml:45]: sink.appliedOps is not latency but number of operations. There seems to be no similar convention in existing entries. "operation" could be a candidate. Do you have any preference?

Comment (everpeace) [reply hbase_regionserver/_metrics.yml:45]: oh, sorry.. my bad.  could you add millis to `ageOfLastAppliedOp` seciton??

Comment (iwasakims) [reply hbase_regionserver/_metrics.yml:45]: sure.

**Comment by everpeace** [comment]
@iwasakims Thanks for your help!!

**Pull request merged**